yt/data_objects/static_output.py

```diff
@@ -572,6 +573,28 @@ def create_field_info(self):
         self.index.field_list = sorted(self.field_list)
         self._last_freq = (None, None)
 
+    def set_field_label_format(self, format_property, value):
+        """
+        Set format properties for how fields will be written
+        out. Accepts 
+
+        format_property : string indicating what property to set
+        value: the value to set for that format_property
+        """
+        available_formats = {"ionizataion_label":("plus_minus", "roman_numeral")}
+        if format_property in available_formats:
+            if value in available_formats[format_property]:
+                setattr(self, "%s_format" % format_property, value)
+            else:
+                raise ValueError("{0} not an acceptable value for format_property \
+                        {1}. Choices are {2}.".format(value, format_property,
+                            available_formats[format_property]))
+        else:
+            raise ValueError("{0} not a recognized format_property. Available
```

Comment:
forgot a continuation character

I usually format multiline strings like this:

```
raise ValueError("this is a really long string "
                 "that i am continuing on the next line.")
```

Which avoids the continuation character.